cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -730,11 +737,127 @@ instance ( ConvertRawHash blk
+                , "prevHashOfBlock" .= String (renderWithOrigin (Text.decodeLatin1 . toRawHash (Proxy @blk)) prevHashOfBlock)
+                ]
+     ImmDB.InvalidChunkFile chunkNo (ImmDB.ChunkErrCorrupt hash blkOrEbb) ->
+       mkObject [ "kind" .= String "TraceImmDBEvent.InvalidChunkFile.InvalidChunkFile"
+                , "chunkNo" .= String (renderChunkNo chunkNo)
+                , "blockOrEBB" .= String (renderBlockOrEBB blkOrEbb)
+                , "hash" .= String (Text.pack $ show hash)
+                ]
+     ImmDB.ChunkFileDoesntFit expectPrevHash actualPrevHash ->
+       mkObject [ "kind" .= String "TraceImmDBEvent.ChunkFileDoesntFit"
+                , "expectedPrevHash" .= String (renderWithOrigin (Text.decodeLatin1 . toRawHash (Proxy @blk)) expectPrevHash)
+                , "actualPrevHash" .= String (renderWithOrigin (Text.decodeLatin1 . toRawHash (Proxy @blk)) actualPrevHash)
+                ]
+     ImmDB.MissingPrimaryIndex chunkNo ->
+       mkObject [ "kind" .= String "TraceImmDBEvent.MissingPrimaryIndex"
+                , "chunkNo" .= String (renderChunkNo chunkNo)
+                ]
+     ImmDB.MissingSecondaryIndex chunkNo ->
+       mkObject [ "kind" .= String "TraceImmDBEvent.MissingSecondaryIndex"
+                , "chunkNo" .= String (renderChunkNo chunkNo)
+                ]
+     ImmDB.InvalidPrimaryIndex chunkNo ->
+       mkObject [ "kind" .= String "TraceImmDBEvent.InvalidPrimaryIndex"
+                , "chunkNo" .= String (renderChunkNo chunkNo)
+                ]
+     ImmDB.InvalidSecondaryIndex chunkNo ->
+       mkObject [ "kind" .= String "TraceImmDBEvent.InvalidSecondaryIndex"
+                , "chunkNo" .= String (renderChunkNo chunkNo)
+                ]
+     ImmDB.RewritePrimaryIndex chunkNo ->
+       mkObject [ "kind" .= String "TraceImmDBEvent.RewritePrimaryIndex"
+                , "chunkNo" .= String (renderChunkNo chunkNo)
+                ]
+     ImmDB.RewriteSecondaryIndex chunkNo ->
+       mkObject [ "kind" .= String "TraceImmDBEvent.RewriteSecondaryIndex"
+                , "chunkNo" .= String (renderChunkNo chunkNo)
+                ]
+     ImmDB.Migrating txt -> mkObject [ "kind" .= String "TraceImmDBEvent.Migrating"
+                                      , "info" .= String txt
+                                      ]
+     ImmDB.DeletingAfter immTipWithInfo ->
+       mkObject [ "kind" .= String "TraceImmDBEvent.DeletingAfter"
+                , "immTipHash" .= String (renderWithOrigin renderTipInfoHash immTipWithInfo)
+                , "immTipBlockNo" .= String (renderWithOrigin renderTipInfoBlockNo immTipWithInfo)
+                ]
+     ImmDB.DBAlreadyClosed -> mkObject [ "kind" .= String "TraceImmDBEvent.DBAlreadyClosed" ]
+     ImmDB.DBClosed -> mkObject [ "kind" .= String "TraceImmDBEvent.DBClosed" ]
+     ImmDB.TraceCacheEvent cacheEv ->
+       case cacheEv of
+         ImmDB.TraceCurrentChunkHit chunkNo noPastChunksinCache ->
```

Comment:
I would rename it to `nbPastChunksInCache` (also capitalise `in`), because I first read it as "no past chunks in cache" (and I even wrote the code in question :slightly_smiling_face:)